apps/etterlatte-vedtaksvurdering/src/main/kotlin/no/nav/etterlatte/vedtaksvurdering/grunnlag/GrunnlagVersjonValidering.kt

```diff
@@ -13,16 +15,27 @@ object GrunnlagVersjonValidering {
         vilkaarsvurdering: VilkaarsvurderingDto?,
         beregningOgAvkorting: BeregningOgAvkorting?,
         trygdetider: List<TrygdetidDto>,
+        behandling: DetaljertBehandling,
     ) {
         logger.info("Sjekker at grunnlagsversjon er konsekvent på tvers av appene")
-
-        if (trygdetider.any { it.opplysningerDifferanse.differanse }) {
-            throw UlikVersjonGrunnlag(
-                "Ulik versjon av grunnlag brukt i trygdetid og behandling." +
-                    "Gå til trygdetidssiden(må kanskje sette vilkårsvurderingen til innvilget) og trykk deretter " +
-                    "'bruk nytt grunnlag'. ",
+        if (vilkaarsvurdering?.resultat?.utfall == VilkaarsvurderingUtfall.IKKE_OPPFYLT &&
+            behandling.boddEllerArbeidetUtlandet?.vurdereAvoededsTrygdeavtale == false
+        ) {
+            logger.warn(
+                "Behandling ${behandling.id} skal ikke ha trygdetid ved avslag da " +
+                    "avdøeds trygdeavtale ikke skal vurderes(ingen avhuking).",
             )
+            // TODO: Vil vi egentlig at vedtak skal slette tt her?
```

Comment:
Jeg tenker at det kanskje ikke er så lurt i akkurat denne metoden, siden dette er en valideringsfunksjon og det virker ille å begynne å slette fra en slik funksjon.

Men det hadde kanskje passet inn i en "rydde"-funksjon? vi kunne evt kalt den fra en river som lytter på attestert og så gjort kall avhengig av vedtaket. Eller ikke

Reply:
Helt enig i det, men jeg er litt sånn kanskje vi skal fjerne det.

Reply:
ja, men tror ikke vi trenger å løse det med en gang - det har allerede skjedd i andre behandlinger vi ikke oppdaget det, der vi ikke har mismatch i trygdetidgrunnlag men vi endret fra innvilgelse til avslag. Og så er det kanskje andre ting vi har lyst til å rydde opp i når vedtak attesteres i tillegg til trygdetiden

Reply:
Ikkesant, virker lurt å ikke lage mye for å støtte noe som kanskje aldri er et problem.